Project.toml

```
@@ -12,7 +12,11 @@ SpecialFunctions = "276daf66-3868-5448-9aa4-cd146d93841b"
 Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
 
 [compat]
-Polynomials = ">= 0.1.0"
+Polynomials = "0.6.0"
```

Is there an issue with using other Polynomials versions in the 0.6.x series (if any exist)? If not, I think this should be
```suggestion
Polynomials = "0.6"
```